index.html

```diff
@@ -3545,13 +3537,27 @@ <h2>
     </section>
     <section class="informative">
       <h2>
-        <code>PaymentRequest</code> and <code>iframe</code> elements
+        Interaction with Feature Policy Specification
       </h2>
-      <p data-tests=
-      "allowpaymentrequest/active-document-cross-origin.https.sub.html, allowpaymentrequest/active-document-same-origin.https.html, allowpaymentrequest/allowpaymentrequest-attribute-cross-origin-bc-containers.https.html, allowpaymentrequest/allowpaymentrequest-attribute-same-origin-bc-containers.https.html, allowpaymentrequest/basic.https.html, allowpaymentrequest/no-attribute-cross-origin-bc-containers.https.html, allowpaymentrequest/no-attribute-same-origin-bc-containers.https.html, allowpaymentrequest/removing-allowpaymentrequest.https.sub.html, allowpaymentrequest/setting-allowpaymentrequest-timing.https.sub.html, allowpaymentrequest/setting-allowpaymentrequest.https.sub.html">
-        To indicate that a cross-origin <a>iframe</a> is allowed to invoke the
-        payment request API, the <a>allowpaymentrequest</a> attribute can be
-        specified on the <a>iframe</a> element.
+      <p>
+        The <a data-cite="feature-policy">Feature Policy</a> specification
+        defines the "<code>payment</code>" feature and the <code><dfn data-cit=
+        "feature-policy#attr-iframe-allowpaymentrequest">allowpaymentrequest</dfn></code>
+        attribute for <a>iframe</a> element, which control whether a particular
```

Comment:
It only defines the processing model for this attribute, the attribute itself is defined by HTML.

Reply:
Makes sense. Will change it back.